### DISCUSSION TITLE
Welcome to Social-Media-Blocks-Extension Discussions!

### DISCUSSION THREAD

**Discussion by JavascriptDon** [JavascriptDon/Social-Media-Blocks-Extension#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tyru5** [comment]
```javascript
console.log('Hello World... erhh.. everyone!👋);
```
My name is Tyrus, however I also go Ti! 👔
I am happy to be part of this community and I appreciate the opportunity to join and contribute to this awesome project.

This is the first time I've really contributed to another `github` repo, and so far it's been awesome! I greatly appreciate everyone's time, and I can't wait to see where we take this. I want to continue contributing to this awesome project / community, and I have a few other ideas I'm going to try out.

Onwards and upwards 🚀

Warmest regards,
Tyrus Malmstrom.

**Reply by JavascriptDon** [reply]
Thank you @Tyru5 for taking the time to make contributions to this project. I love the contributions you made to this project and I look forward to seeing your future suggestions! 🤓

Kind regards,
Helitha

**Reply by Tyru5** [reply]
Thanks a bunch @JavascriptDon , greatly appreciate it! 😃

**Comment by LucidSamuel** [comment]
Hello all, 
My name is Samuel and I'm glad to be part of this community,
I'm currently looking forward to contributing with a little landing page (if we all think it's going to be useful) basically to give users the ability to see what the project is about and download the extension with more ease.

**Reply by JavascriptDon** [reply]
Hey @LucidSamuel ! Thank you for taking the time to help me out.

**Reply by Tyru5** [reply]
Hey @LucidSamuel, nice to meet you, and happy to hear you are part of this community!

And I love the idea of a landing page for this Chrome extension.
